src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -45,6 +45,10 @@
                     >
                     </label>
                 </tree-select>
+                <div v-if="fetchingDatasets" class="pt-2" id="fetching-datasets">
+                    <loading-spinner  size="xs"></loading-spinner>
```

Comment:
I hadn't realised adding a loading spinner would be this straightforward. My illusion that the speed the spinner spins gives any indication about what is going on under the hood has been shattered!